src/components/DownloadAndShareComponents/hooks/useResourceFormSubmit.ts

```diff
@@ -60,6 +60,13 @@ const useResourceFormSubmit = (props: UseResourceFormProps) => {
       setTermsInLocalStorage(terms);
     }
     if (props.type === "download") {
+      await downloadLessonResources(
+        slug,
+        downloads as DownloadResourceType[],
+        props.isLegacyDownload,
+      );
+    }
+    if (props.type === "share" || props.type === "download") {
```

Comment:
does this need to be in an if? aren't they the only options?

Reply:
Yep doesn't need to be there